allennlp/data/dataset_readers/dataset_utils/span_utils.py

```diff
@@ -0,0 +1,125 @@
+from typing import List, Tuple, Callable
+
+from allennlp.data.dataset_readers.dataset_utils.ontonotes import TypedStringSpan
+from allennlp.data.tokenizers.token import Token
+
+def enumerate_spans(sentence: List[Token],
+                    offset: int = 0,
+                    max_span_width: int = None,
+                    min_span_width: int = None,
+                    filter_function: Callable[[List[Token]], bool] = None) -> List[Tuple[int, int]]:
```

Comment:
This parameter isn't documented.  It'd be nice to explicitly state somewhere that returning `True` from this function means "keep the span" - that's not always obvious from the name "filter".